src/cache/cache.rs

```diff
@@ -397,6 +397,18 @@ impl Storage for opendal::Operator {
             Ok(_) => (),
             // Read not exist file with not found is ok.
             Err(err) if err.kind() == ErrorKind::ObjectNotFound => (),
+            // Tricky Part.
+            //
+            // We tolerate rate limited here to make sccache keep running.
```

Comment:
maybe document that in the gha doc ?

something like "in case sccache reaches the rate limit of the service, the build will continue but the storage might not be performed"

Reply:
Added.